common/src/main/java/com/wynntils/features/user/DurabilityArcFeature.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.config.properties.Configurable;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.EventListener;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.features.properties.FeatureInfo.Stability;
+import com.wynntils.mc.event.SlotRenderEvent;
+import com.wynntils.mc.render.RenderUtils;
+import com.wynntils.utils.objects.CustomColor;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import com.wynntils.wc.custom.item.properties.DurabilityProperty;
+import com.wynntils.wc.custom.item.properties.ItemProperty;
+import net.minecraft.util.Mth;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@EventListener
+@FeatureInfo(stability = Stability.STABLE)
+@Configurable(category = "Inventory")
+public class DurabilityArcFeature extends UserFeature {
+
+    @SubscribeEvent
+    public void onRenderSlot(SlotRenderEvent.Pre e) {
+        ItemStack item = e.getSlot().getItem();
+        if (!(item instanceof WynnItemStack wynnItem)) return;
+
+        if (!wynnItem.hasProperty(ItemProperty.DURABILITY)) return; // no durability info
+        DurabilityProperty durability = wynnItem.getProperty(ItemProperty.DURABILITY);
+
+        // calculate color of arc
+        float durabilityPct = durability.getDurabilityPct();
```

Comment:
Maybe please spell out "percent"? It took me a while to figure out that it was not a "picture" this was about... (Both the local var and the method)